CHANGELOG.md

```diff
@@ -1,5 +1,89 @@
 # Changelog
 
+## 0.14 (2021-02-01)
+
+#### :sparkles: `Loader` interface for parsing environments
+
+While introducing inline environments in the last release, the parsing logic became harder
+to understand and maintain. To improve the situation, it received a major overhaul so we
+can handle spec.json-based and inline environments more efficiently.
```

Comment:
This is not a user visible change, let's feature something more cool as the first thing in the changelog

Reply:
I dropped it, feel free to make more concrete suggestions.